src/lib/es5.d.ts

```diff
@@ -678,7 +678,7 @@ interface Math {
      */
     atan(x: number): number;
     /**
-     * Returns the angle (in radians) from the X axis to a point.
+     * Returns the angle (in radians) between the X axis and a line going through the given point.

```

Comment:
"a line" could be anything.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/atan2 seems more accurate

Reply:
Done.